.stylelintrc.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "stylelint-config-sass-guidelines",
+  "rules": {
```

Comment:
qua ti direi di usare anche il plugin stylelint-scss che aggiunge un set di regole specifiche a scss:

https://github.com/kristerkari/stylelint-scss#installation-and-usage

Reply:
scherzavo; ho visto ora che è incluso dal tuo extends. grande!